scipy/interpolate/fitpack2.py

```diff
@@ -1579,6 +1577,17 @@ def __init__(self, theta, phi, r, w=None, s=0., eps=1E-16):
         self.tck = tt_[:nt_], tp_[:np_], c[:(nt_ - 4) * (np_ - 4)]
         self.degrees = (3, 3)
 
+    def __call__(self, theta, phi, dtheta=0, dphi=0, grid=True):
+
+        theta = np.asarray(theta)
+        phi = np.asarray(phi)
+
+        if phi.size > 0 and (phi.min() < 0. or phi.max() > 2. * np.pi):
+            raise ValueError("requested phi out of bounds.")
+
+        return SphereBivariateSpline.__call__(self, theta, phi, dtheta=dtheta,
+                                       dphi=dphi, grid=grid)
```

Comment:
minor: indentation is off here, can you line up `dphi` with `self`?

Reply:
and same further down

Reply:
Thank you I fixed it.